client/src/components/settings/Username.js

```diff
@@ -60,7 +60,8 @@ class UsernameSettings extends Component {
       isFormPristine: true,
       formValue: props.username,
       characterValidation: { valid: false, error: null },
-      submitClicked: false
+      submitClicked: false,
+      isUserNew: true
```

Comment:
```suggestion
      isUserNew: /^fcc[0-9|a-f]{8}-[0-9|a-f]{4}-[0-9|a-f]{4}-[0-9|a-f]{4}-[0-9|a-f]{12}$/.test(
          prevUsername
        )
```
The test should also go here, so that it deals with freshly reloaded pages.